config/locales/en.yml

```diff
@@ -796,7 +797,22 @@ en:
     order_receipt:
       subject: "NU Core Order Receipt"
       intro: "Thank you for your order on the NU Core website."
+      label:
+        ordered_at: Ordered at
+        facility: Facility
+        ordered_by: Ordered By
+      order_details: Order Details
       outro: "All prices are estimates. Actual cost is assigned when the order is complete."
+      reservation: Reservation
+      th:
+        order_number: "Order #"
+        product: Product
+        quantity: Quantity
+        status: Status
+        price: "%{text}Price"
```

Comment:
This is for the text version of the receipt. The way it was working was the "text" piece coming out of `order_estimated_or_actual_text` might be a prefix like "Estimated", or an empty string. The extra space wasn't so much a problem in the html version (though I like that it's gone in this patch).